recipes/openssh/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import copy, get, rmdir, export_conandata_patches
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+from conan.tools.layout import basic_layout
+from os.path import join
+
+required_conan_version = ">=1.54.0"
+
+
+class PackageConan(ConanFile):
+    name = "openssh"
+    description = "The OpenSSH (portable) suite of secure connectivity tools"
+    license = "SSH-OpenSSH"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.openssh.com/portable.html"
+    topics = ("security", "cryptography", "login", "keychain", "file-sharing", "ssh")
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_libcrypto": [None, "libressl", "openssl"],
+        "with_pam": [None, "openpam"],
+        "with_selinux": [True, False],
+        "with_libedit": [True, False],
+        "with_sandbox": ["auto", "no", "capsicum", "darwin", "rlimit", "seccomp_filter", "systrace", "pledge"]
+    }
+    default_options = {
+        "with_libcrypto": "openssl",
+        "with_pam": None,
+        "with_selinux": False,
+        "with_libedit": False,
+        "with_sandbox": "auto"
+    }
+
+    def package_id(self):
+        del self.info.settings.compiler
+        del self.info.settings.build_type
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def configure(self):
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("zlib/[>=1.2.11 <2]")
+        if self.options.with_libcrypto  == "openssl":
+            self.requires("openssl/[>=1.1 <=3.1]")
+        elif self.options.with_libcrypto  == "libressl":
+            self.requires("libressl/[>=2.9 <=3.5]")
```

Comment:
```suggestion
            self.requires("libressl/3.5.3")
```

Assuming the original upper limit meant <3.6 (Else 3.5 would be in, but 3.5.1 wouldn't)

I'm not cnfident in the backwards compatibility of libress to allow version ranges for its version, so pinning for the latest available one currently